tests/utils/test_tasks.py

```diff
@@ -0,0 +1,24 @@
+import os
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from arches.app.models import models
+from arches.app.tasks import package_load_complete
+
+# these tests can be run from the command line via
+# python manage.py test tests.utils.test_tasks.TaskTests --settings="tests.test_settings"
+
+
+class TaskTests(TestCase):
+    def test_package_load_complete(self):
+        resource_path = os.path.join("tests", "fixtures", "data", "json", "example_source_business_data.json")
+        with patch("logging.Logger.warning") as mock_warning:
+            # This used to emit a warning: "Error occurred sending email." 
```

Comment:
Yeah, I was actually trying to bail out before the logged warning for celery import tasks, since they aren't supposed to send emails, and I found the log confusing. WDYT?

Reply:
Looks like I had this a little wrong 💡  -- the package load notification does involve an email according to the NotificationType record (if your admin user has an email recorded), so there's a bug I'll patch to make it do that.